src/uterm_drm_shared.c

```diff
@@ -599,6 +599,10 @@ static void bind_display(struct uterm_video *video, drmModeRes *res,
 		return;
 	ddrm = disp->data;
 
+	drmModeEncoder *enc = drmModeGetEncoder(vdrm->fd, conn->encoder_id);
+	drmModeCrtc *current_crtc = drmModeGetCrtc(vdrm->fd, enc->crtc_id);
```

Comment:
This causes a segfault on my two test PC.
enc is NULL, and using enc->crtc_id leads to a crash.